src/category_theory/groupoid/free_groupoid.lean

```diff
@@ -170,14 +170,16 @@ lemma lift_unique (φ : prefunctor V V') (Φ : free_groupoid V ⥤ V')
 begin
   apply quotient.lift_unique,
   apply paths.lift_unique,
-  apply quiver.symmetrify.lift_unique,
-  { rw ←functor.to_prefunctor_comp, exact hΦ, },
-  { rintros X Y f,
+  haveI : (paths.of ≫q (quotient.functor red_step ⋙ Φ).to_prefunctor).preserves_reverse, by
+  { constructor, rintros X Y f,
```

Comment:
Why is this an improvement?

Reply:
Well, now there is a class for `preserves_reverse`, so I needed an `haveI`

Reply:
Oh, I see.

I would have just used `apply @quiver.symmetrify.lift_unique _ _ _ _ _ _ _ _ _,`, to force creation of the instance as a goal, rather than having to write out the obscure type in `haveI`. But I guess it's okay.

Reply:
(My complaint was just that it seemed you had restructured the proof in such a way that you had to write out an obscure type, which seemed a step backwards.)

Reply:
Ah, I hadn't thought of doing it this way, using `apply`. I agree it's clearer though.